app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt

```diff
@@ -2,11 +2,19 @@ package com.theopensourcefamily.chessclock
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.jakewharton.rxbinding3.view.clicks
+import io.reactivex.Observable
 import kotlinx.android.synthetic.main.activity_clocks.*
 
-class ClocksActivity : AppCompatActivity(), GameView {
+class ClocksActivity : AppCompatActivity(), ClocksView {
 
-  val presenter = GamePresenter() // maybe DI here, with koin or wathever
+  val presenter = ClocksPresenter() // maybe DI here, with koin or wathever
```

Comment:
Great idea! What do you think about adding this here or in another PR, I don't care.

Reply:
Yes I implemented this with Koin, but then i was thinking that maybe we don't ever need an instance of the presenter in the activity... and I'm not sure how to do this "scoped" injection with Koin and I didn't want to get stuck with this in this PR 😅  
We can create an issue just to do this! and this issue wont block any further implementations 😋

Reply:
Ok! I agree! :)